packages/sequelize-data-provider/src/sequelizeDataProvider.test.ts

```diff
@@ -531,6 +532,168 @@ describe('sequelizeDataProvider', () => {
+        expect(relatedLobbyists1).toEqual([]);
+        expect(relatedMinistries2).toEqual([]);
+      });
+
+      test('loadRelatedEntities multiple keys with different args', async () => {
+        const lobbyist1 = await sequelizeDataProvider.createEntity('Lobbyist', {
+          name: `lobbyist1${randomNumber}`,
+          governments: {
+            connect: [
+              {
+                id: government1.id
+              }
+            ]
+          }
+        });
+        const [relatedMinistries1a, relatedMinistries1b, relatedLobbyists1, relatedMinistries2] = await loadRelatedEntities(
+          'Government',
+          [
+            { originalEntityId: government1.id, relationEntityName: 'ministries', args: { where: { name: ministry2.name } } },
+            { originalEntityId: government1.id, relationEntityName: 'ministries', args: { where: { name_not: ministry1.name } } },
+            { originalEntityId: government1.id, relationEntityName: 'lobbyists' },
+            { originalEntityId: government1.id, relationEntityName: 'ministries' }
+          ],
+          sequelizeDataProvider.getRelatedEntities
+        );
+        expect(relatedMinistries1a.map(g => g.id).sort()).toEqual([ministry2.id].sort());
+        expect(relatedMinistries1b.map(g => g.id).sort()).toEqual([ministry2.id, ministry3.id].sort());
+        expect(relatedLobbyists1.map(g => g.id)).toEqual([lobbyist1.id]);
+        expect(relatedMinistries2.map(g => g.id).sort()).toEqual([ministry1.id, ministry2.id, ministry3.id].sort());
+        await sequelizeDataProvider.updateEntity(
+          'Lobbyist',
+          {
+            governments: {
+              disconnect: [
+                {
+                  id: government1.id
+                }
+              ]
+            }
+          },
+          {
+            id: lobbyist1.id
+          }
+        );
+        await sequelizeDataProvider.deleteEntity('Lobbyist', { id: lobbyist1.id });
+      });
+    });
+
+    test('getRelatedEntityIds no args', async () => {
+      const fetchedMinistryIds = await sequelizeDataProvider.getRelatedEntityIds('Government', government1.id, 'ministries');
+      expect(fetchedMinistryIds.sort()).toEqual([ministry1.id, ministry2.id, ministry3.id].sort());
+    });
+
+    test('getRelatedEntityIds with args', async () => {
+      const fetchedMinistryIds = await sequelizeDataProvider.getRelatedEntityIds('Government', government1.id, 'ministries', {
+        first: 1,
+        orderBy: 'createdAt_ASC'
+      });
+      expect(fetchedMinistryIds).toHaveLength(1);
+      expect(fetchedMinistryIds[0]).toEqual(ministry1.id);
+    });
+
+    test('getRelatedEntityIds with args for nXm relation', async () => {
+      const fetchedVoteIds = await sequelizeDataProvider.getRelatedEntityIds('Minister', minister2.id, 'votes', {
+        first: 1,
+        orderBy: 'createdAt_ASC'
+      });
+      expect(fetchedVoteIds).toHaveLength(1);
+    });
```

Comment:
This whole chunk is duplicated in lines 512-532

Reply:
yeah, a copy paste i forgot to delete (;